apps/annotations/models.py

```diff
@@ -98,6 +100,7 @@ class Meta:
         abstract = True
 
     tags = TaggableManager(through=CustomTaggedItem)
+    _skipped_category_tags: ClassVar[list] = [TagCategories.MEDIA_TYPE.value]  # Tag categories with special treatment
```

Comment:
I still think this [comment is](https://github.com/dimagi/open-chat-studio/pull/2567/changes#r2618539957) valid. Calling it `_skipped_category_tags` does not indicate why these needs to be skipped. Skipped from what? It's not clear unless you first look at where / how it's being used. Basically this is what I'm trying to solve